restAPI/routes/route_auth.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+// Controller
+const controller_appraisal_modules = require("../controllers/controller_auth");
+
+router.get("/test", controller_appraisal_modules.testAPI);
+router.post("/login", controller_appraisal_modules.login);
```

Comment:
```suggestion
router.post("/login", controller_appraisal_modules.loginAPI);
```
The same convention should be followed across the application.